ews-java-api.iml

```diff
@@ -1,6 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="false">
+  <component name="FacetManager">
+    <facet type="Osmorc" name="OSGi">
+      <configuration manifestGenerationMode="OsmorcControlled" manifestLocation="" jarfileLocation="ews-java-api-1.3-SNAPSHOT.jar" outputPathType="CompilerOutputPath" bndFileLocation="" bundlorFileLocation="" bundleActivator="" bundleSymbolicName="com.microsoft.office.ews-java-api" bundleVersion="1.3.0.SNAPSHOT" ignoreFilePattern="" useProjectDefaultManifestFileLocation="true" alwaysRebuildBundleJAR="false" doNotSynchronizeWithMaven="false">
+        <additionalProperties>
+          <property key="Bundle-Description" value="Exchange Web Services (EWS) Java API" />
+          <property key="Bundle-Vendor" value="Microsoft" />
+          <property key="Bundle-DocURL" value="http://www.microsoft.com/" />
+        </additionalProperties>
+        <additionalJARContents />
+      </configuration>
+    </facet>
+  </component>
```

Comment:
Then it will be best to update this file after every version change, I guess. Another option would be to put this file in `.gitignore` and add this file with force when it needs updating. Otherwise every contributor will need to exclude this file from new commits, as IntelliJ will keep adding these lines.

The option of adding this file to `.gitignore` is a dangerous one, I think, as it won't be clear when the file does need updating in the repository.